tls/dub.sdl

```diff
@@ -26,44 +26,60 @@ configuration "openssl" {
 	sourceFiles "../lib/win-i386/libssl.lib" "../lib/win-i386/libcrypto.lib" platform="windows-x86-dmd"
 
 	// Posix
-	preGenerateCommands `rdmd --eval='
-	auto dir = environment.get("DUB_PACKAGE_DIR");
-	if (dir.buildPath("tls").exists)  {
-		dir = dir.buildPath("tls");
-	}
-	string opensslVersion;
-	try {
-		const res = execute(["pkg-config", "openssl", "--modversion"]);
-		if (res.status == 0)
-			opensslVersion = res.output.strip();
-	} catch (Exception e) {}
-
-	if (!opensslVersion.length) try
+	preGenerateCommands `echo '
+	/+ dub.sdl:
+	name "script"
+	+/
+	import std.process;
+	import std.stdio;
+	import std.string;
+	import std.range;
+	import std.uni;
+	import std.file;
+	import std.conv;
+	import std.algorithm;
+	import std.path;
+    import std.functional;
```

Comment:
Indent is off. Also can you order those ?

Reply:
Fixed